lib/fixtures/challenges-table.sql

```diff
@@ -16,6 +16,6 @@ CREATE TABLE IF NOT EXISTS challenges (
 
 -- Create table
 CREATE TABLE IF NOT EXISTS challenges_tags (
-  tag_id INTEGER REFERENCES tags (id),
-  challenge_id INTEGER REFERENCES challenges (id)
+  challenge_id INTEGER REFERENCES challenges (id),
```

Comment:
I think the convention is to challenges_id and tags_id now, is that right @JMurphyWeb?